### PR TITLE
Display token on my account token overlay

### LINKED ISSUE #2285: Add the ability to display the token in the UI
as a new user
with admin permissions or higher
when i view the list of tokens
then i would like the ability to view the text of the token
then i would like the ability to copy the text of the token to my clipboard

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #2285

Extracted copy to clipboard button from onboarding to it's separate component which takes in token or any snippet to be copied. CopyToken component also takes in notify prop to show notification that the text was copied.

feature-2285

### CODE REVIEW

File: ui/src/onboarding/components/verifyStep/TelegrafInstructions.tsx [hunk 1]
```diff
@@ -47,61 +41,13 @@ class TelegrafInstructions extends PureComponent<Props> {
...
-                customClass="wizard-step--body-copybutton"
-                size={ComponentSize.Small}
-                color={ComponentColor.Default}
-                titleText="copy to clipboard"
-                text="Copy"
-                onClick={this.handleClickCopy}
-              />
-            </CopyToClipboard>
-          </p>
+          <CopyText copyText={configScript} notify={notify} />
```
Comment (OfTheDelmer) [inline ui/src/onboarding/components/verifyStep/TelegrafInstructions.tsx]: This component really helped remove a lot of boilerplate

File: ui/src/shared/components/CopyText.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// Actions
+import {NotificationAction} from 'src/types'
+import {
+  copyToClipboardSuccess,
+  copyToClipboardFailed,
+} from 'src/shared/copy/notifications'
+
+export interface Props {
+  copyText: string
+  notify?: NotificationAction
```
Comment (OfTheDelmer) [inline ui/src/shared/components/CopyText.tsx]: Do we want this to be optional?

File: ui/src/shared/components/CopyText.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  private handleClickCopy = (e: MouseEvent<HTMLButtonElement>) => {
+    e.stopPropagation()
+    e.preventDefault()
+  }
+
+  private handleCopyAttempt = (
+    copiedText: string,
+    isSuccessful: boolean
+  ): void => {
+    const {notify} = this.props
```
Comment (OfTheDelmer) [inline ui/src/shared/components/CopyText.tsx]: We may want a guard here if `notify` is not provided.

File: ui/src/shared/components/CopyText.tsx (hunk 2 above)
Comment (Palakp41) [reply ui/src/shared/components/CopyText.tsx]: Actually I will make it required prop. It makes sense to see a notification when text is copied.

File: ui/src/onboarding/components/verifyStep/TelegrafInstructions.tsx (hunk 1 above)
Comment (Palakp41) [reply ui/src/onboarding/components/verifyStep/TelegrafInstructions.tsx]: Yes, it did :)

**Pull request merged**